### PR TITLE
Moved service to another module

### PR DESCRIPTION
**Pull request by Yura09**

### CODE REVIEW

**Review by eyouu — APPROVED** [review]

File: service-api/pom.xml [hunk 1]
```diff
@@ -13,12 +13,67 @@
     <artifactId>service-api</artifactId>
     <version>1.0-SNAPSHOT</version>
     <name>service-api</name>
-
+    <properties>
+        <junit-jupiter.version>5.4.2</junit-jupiter.version>
+        <mockito-junit-jupiter.version>3.3.3</mockito-junit-jupiter.version>
```
Comment (antonbakalets) [inline service-api/pom.xml:18]: Please move properties into parent pom.xml.

**Review by antonbakalets — COMMENTED** [review]
Please hold down with this task:

The `...ServiceImpl` and `...ServiceImplTest` should be in `service` module, not `service-api`

File: core/pom.xml [hunk 2]
```diff
@@ -32,6 +28,18 @@
...
+            <groupId>greencity</groupId>
+            <artifactId>service-api</artifactId>
+            <version>${project.version}</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>greencity</groupId>
+            <artifactId>service</artifactId>
+            <version>${project.version}</version>
+            <scope>runtime</scope>
```
Comment (antonbakalets) [inline core/pom.xml:41]: Please comment this out and mark as TODO in #1618

File: service-api/pom.xml [hunk 3]
```diff
@@ -20,5 +20,20 @@
             <artifactId>lombok</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
```
Comment (antonbakalets) [inline service-api/pom.xml:24]: Why do you need this? I believe you don't need the whole web, but just some of dependencies from it.

File: service-api/pom.xml [hunk 4]
```diff
@@ -20,5 +20,20 @@
...
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>javax.validation</groupId>
+            <artifactId>validation-api</artifactId>
+            <version>2.0.1.Final</version>
+        </dependency>
+        <dependency>
+            <groupId>greencity</groupId>
```
Comment (antonbakalets) [inline service-api/pom.xml:33]: This must be in Service-Impl module. The API don't based on what the implementation is done.

File: service-api/pom.xml [hunk 5]
```diff
@@ -20,5 +20,20 @@
...
+            <artifactId>dao</artifactId>
+            <version>1.0-SNAPSHOT</version>
+            <scope>runtime</scope>
```
Comment (antonbakalets) [inline service-api/pom.xml:36]: Please comment this out and mark as TODO in #1618

File: service-api/src/main/java/greencity/constant/CacheConstants.java [hunk 6]
```diff
@@ -0,0 +1,11 @@
+package greencity.constant;
+
+/**
+ * Class, that contains constants used for caching.
+ */
+public final class CacheConstants {
+    public static final String ALL_ACHIEVEMENTS_CACHE_NAME = "achievements_cache";
```
Comment (antonbakalets) [inline service-api/src/main/java/greencity/constant/CacheConstants.java:7]: move to Impl.

File: service/pom.xml [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+    <groupId>greencity</groupId>
+    <artifactId>service</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <name>service</name>
+    <dependencies>
+        <dependency>
+            <groupId>greencity</groupId>
+            <artifactId>dao</artifactId>
+            <version>${project.version}</version>
+            <scope>compile</scope>
```
Comment (antonbakalets) [inline service/pom.xml:21]: runtime, commented and marked as TODO

File: service/pom.xml [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+            <artifactId>lombok</artifactId>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>javax.validation</groupId>
+            <artifactId>validation-api</artifactId>
```
Comment (antonbakalets) [inline service/pom.xml:40]: Already available from Service-API pom

**Review by antonbakalets — CHANGES_REQUESTED** [review]

**Review by antonbakalets — APPROVED** [review]

**Review by forestik — APPROVED** [review]

**Pull request merged**